docs/troubleshoot/app-framework/zis-error-codes.md

```diff
@@ -0,0 +1,1985 @@
+
+One of the callbacks of a plug-in failed.
+
+**Action:**
+
+Depending on the stage, some of the plug-in functionality might be impacted.
+Contact support if you cannot resolve the issue.
+
+### ZWES0018W
+
+Plug-in '_plug-in_name_' version _plug-in_version_ doesn't match anchor version
+_anchor_version_, LPA module discarded
+
+**Reason:**
+
+The cross-memory server detected that a plug-in module was outdated; this
+usually happens when a plug-in gets updated.
+
+**Action:**
+
+The cross-memory server discards the old module and loads the latest version to
+the link pack area (LPA).
+
+### ZWES0019W
+
+Parameter '_parameter_name_' has an invalid value
+
+**Reason:**
+
+The cross-memory server detected an invalid parameter.
+
+**Action:**
+
+The cross-memory server uses the default parameter. Fix the reported parameter
+and restart the cross-memory server.
+
+### ZWES0020E
+
+ZSS Cross-Memory server PARMLIB member name not determined, RC = _return_code_
+
+**Reason:**
+
+The cross-memory server could not determine which PARMLIB member to use.
+
+**Action:**
+
+The cross-memory server terminates. Contact support.
+
+### ZWES0021E
+
+ZSS Cross-Memory server module member name not determined, RC =
+_csvquery_return_code_
+
+**Reason:**
+
+The cross-memory server could not determine its module name.
+
+**Action:**
+
+The cross-memory server terminates. Review the provided CSVQUERY return code (
+see "z/OS MVS Programming: Assembler Services Reference ABE-HSP") and contact
+support if you cannot resolve the issue.
+
+### ZWES0098I
+
+_debug_message_
+
+**Reason:**
+
+This is a debug message.
+
+**Action:**
+
+No action required.
+
+### ZWES0099I
+
+_hex_dump_
+
+**Reason:**
+
+This is a debug hex dump.
+
+**Action:**
+
+No action required.
+
+## ZIS Auxiliary Server messages
+
+### ZWES0050I
+
+ZIS AUX Server starting, version is _major_._minor_._patch_+_datestamp_
+
+**Reason:**
+
+The cross-memory auxiliary server with the specified version is starting.
+
+**Action:**
+
+No action required.
+
+### ZWES0051I
+
+ZIS AUX Server terminated
+
+**Reason:**
+
+The cross-memory auxiliary server fully terminated.
+
+**Action:**
+
+No action required.
+
+### ZWES0052I
+
+Input parameters at _address_:
+
+**Reason:**
+
+The message shows a dump of the parameters passed to this address space.
+
+**Action:**
+
+No action required.
+
+### ZWES0053E
+
+Not APF-authorized (_testauth_status_)
+
+**Reason:**
+
+One or more data sets in the STEPLIB concatenation is not APF-authorized.
+
+**Action:**
+
+The cross-memory auxiliary server terminates. Ensure that all the STEPLIB data
+sets are APF-authorized.
+
+### ZWES0054E
+
+ZIS AUX Server started in wrong key _key_
+
+**Reason:**
+
+The cross-memory auxiliary server detected that it was running in the wrong key.
+
+**Action:**
+
+The cross-memory auxiliary server terminates. Ensure that you have added the
+correct PPT-entry (
+see [the documentation](https://docs.zowe.org/stable/user-guide/configure-xmem-server/))
```

Comment:
These types of URLs could break if someone changes the doc tech.
Instead, try relative:
(../../user-guide/configure-xmem-server.md)

you will see examples of this elsewhere, https://github.com/ifakhrutdinov/docs-site/blob/feature/zis-message-ids/docs/user-guide/systemrequirements-zos.md?plain=1#L99

Reply:
Fixed.